src/components/body/BodyMainPanel.tsx

```diff
@@ -45,20 +57,82 @@ const BodyMainPanel: React.FC<IProps> = ({
 
   const contentContainerStyle = React.useMemo(
     () => ({
-      paddingLeft: styleLeft,
-      paddingTop: styleTop,
+      top: styleTop,
+      left: styleLeft,
+      paddingLeft: _scrollLeft,
+      paddingTop: _scrollTop,
       height: dataLength * bodyRowHeight,
```

Comment:
```suggestion
      height: bodyCotentHeight
```
Fix the height value to Memo